lib/decanter/value_parser/datetime_parser.rb

```diff
@@ -5,8 +5,8 @@ class DateTimeParser < Base
       allow DateTime
 
       parser do |name, val, options|
-        parse_format = options.fetch(:parse_format, '%m/%d/%Y')
-        ::Date.strptime(val, parse_format)
+        parse_format = options.fetch(:parse_format, '%m/%d/%Y %I:%M:%S %p')
+        ::DateTime.strptime(val, parse_format)
```

Comment:
One thing to keep in mind is that any apps that are currently using ```DateTimeParser``` to parse a string like "05/18/2015" will need to switch to use ```DateParser``` instead as this would cause an "Invalid Date" error.